api/serializers/canvas_course_registration.py

```diff
@@ -29,4 +29,4 @@ def get_is_registered(self, course):
     class Meta:
         model = CanvasCourseRegistration
         fields = ['id', 'canvas_user_id', 'is_verified', 'is_blocked', 'token_uses', 'total_tokens_received',
-                  'available_tokens', 'username', 'name']
```

Comment:
I believe these changes are already in master. Please pull from master and avoid duplicated code.